codegens/nodejs-axios/lib/axios.js

```diff
@@ -0,0 +1,225 @@
+      if (type === 'file') {
+        // if src is not of type string we check for array(multiple files)
+        if (typeof param.src !== 'string') {
+          // if src is an array(not empty), iterate over it and add files as separate form fields
+          if (Array.isArray(param.src) && param.src.length) {
+            param.src.forEach((filePath) => {
+              addFormParam(formdataArray, key, param.type, filePath, disabled, contentType);
+            });
+          }
+          // if src is not an array or string, or is an empty array, add a placeholder for file path(no files case)
+          else {
+            addFormParam(formdataArray, key, param.type, '/path/to/file', disabled, contentType);
+          }
+        }
+        // if src is string, directly add the param with src as filepath
+        else {
+          addFormParam(formdataArray, key, param.type, param.src, disabled, contentType);
+        }
+      }
+      // if type is text, directly add it to formdata array
+      else {
+        addFormParam(formdataArray, key, param.type, param.value, disabled, contentType);
+      }
+    });
+
+    request.body.update({
+      mode: 'formdata',
+      formdata: formdataArray
+    });
+  }
+  
+  body = request.body && request.body.toJSON();
+    
+  dataSnippet = parseRequest.parseBody(body, 
+                                    options.trimRequestBody,
+                                    indent,
+                                    request.headers.get('Content-Type'),
+                                    options.ES6_enabled);
+  snippet += dataSnippet+'\n';
+
+  configArray.push(indentString + `'method': '${request.method.toLowerCase()}'`);
+  configArray.push(indentString + `'url': '${sanitize(request.url.toString())}'`);  
+  
+  headers = parseRequest.parseHeader(request, indentString)
+  // https://github.com/axios/axios/issues/789#issuecomment-577177492
+  if (!_.isEmpty(body) && body.formdata ) {
+    // we can assume that data object is filled up
+    headers.push(`\n${indentString}...data.getHeaders()`);
+  }
+  let headerSnippet = indentString + "'headers': { " 
+  if(headers.length >0 ) {
+    headerSnippet += "\n"
+    headerSnippet += indentString+ headers.join(', \n'+indentString) + "\n"
+    headerSnippet += indentString + '}'
+  } else {
+    headerSnippet += '}'
+  }
+  
+  configArray.push(headerSnippet);
+  
+  if (options.requestTimeout) {
+    configArray.push(indentString + `timeout: ${options.requestTimeout}`);
+  }
+  if (options.followRedirect === false) {
+    // setting the maxRedirects to 0 will disable any redirects.
+    // by default, maxRedirects are set to 5
+    configArray.push(indentString + 'maxRedirects: 0');
+  }
+  if (dataSnippet !== '') {
+    // although just data is enough, whatever :shrug:
+    configArray.push(indentString +`data : data`)
+  }
+  snippet += varDeclare + ' config = {\n';
+  snippet += configArray.join(',\n') + '\n';
+  snippet += '}\n';
```

Comment:
need a semicolon and a new line after the closing bracket.

Reply:
done